publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
+These answers are generally written to be applicable to all WoT deliverables.
+Where a response is specifically relevant to a particular deliverable that will be identified.
+
+## If technology allows visual rendering of content
+The WoT architecture does not directly support visual rendering of content.
```

Comment:
Suggest to make it more clear, that WoT is describing protocol level interfaces and not user interfaces of devices. 
It is possible to auto-generate a baseline UI for the elements of this interface description, however WoT does not target describing user interfaces. 
Therefore many of the questions of this questionnaire are not applicable.

Reply:
I have added a couple of introductory paragraphs to set the scope and define how we are re-interpreting the questions.